src/internal/util/isObservable.ts

```diff
@@ -1,9 +1,11 @@
+/** @prettier */
 import { Observable } from '../Observable';
+import { isFunction } from './isFunction';
 
 /**
  * Tests to see if the object is an RxJS {@link Observable}
  * @param obj the object to test
  */
 export function isObservable<T>(obj: any): obj is Observable<T> {
-  return !!obj && (obj instanceof Observable || (typeof obj.lift === 'function' && typeof obj.subscribe === 'function'));
+  return !!obj && (obj instanceof Observable || (isFunction(obj.lift) && isFunction(obj.subscribe)));
```

Comment:
```suggestion
  return obj && (obj instanceof Observable || (isFunction(obj.lift) && isFunction(obj.subscribe)));
```

Reply:
So this change actually illustrates why I'm not always a fan of the inline comment changes. I looked and thought "yeah, that's reasonable" and merged it, and it broke a test :)

I've reverted that change and added a comment. But basically, this is a publicly exposed API, and we want to make sure it returns a boolean.... but if someone passes `null`, it would return `null` and not `false` after this change.